_posts/2024-01-17-devops-and-flutterflow.markdown

```diff
@@ -0,0 +1,52 @@
+---
+layout: postv2
+font: serif
+title: "Role of FlutterFlow in Modern App Development: Insights for DevOps Teams"
+description: "FlutterFlow is a visual editor for building Flutter apps. It is a good starting point to move github and use DevOps practices to build out a full app"
+date: 2023-10-17
+highlight: monokai
+image: img/devopsflutterflow/cover.png
+author: Matthew Carter
+tags: [Trust policy, GitHub, GitHub Environments, Security, OIDC]
+---
+
+# Role of FlutterFlow in Modern App Development: Insights for DevOps Teams
+
+## Introduction
+
+- A click and drag interface for building basic Flutter apps quickly
+- Good base for building out a POC or MVP
+- Can be used to build out a full app
+
+## What is FlutterFlow?
+
+- A visual editor for building Flutter apps
+- Extensive templates and pre-built components
+- Database integrations
+- Authentication
+- Export to code
+
+## Beyond FlutterFlow
+
+- FlutterFlow is a good starting point but you can push the entire code base to github
+
+## DevOps and FlutterFlow
+
+- Automated Testing
+  - Flutter Integration tests and firebase test lab
+  - Need a pro plan
+- Version control
+  - Can ClickOps it but better to use git
+- CI/CD & Trunk Based Development
+  - Not possible to enforces testing before publishing
```

Comment:
interesting - seems like a big gap

Reply:
Might be possible but can't see it being possible yet - going to try the pro version free trail to test this and a couple other things.